Client/src/Pages/Auth/Register/StepThree/index.jsx

```
@@ -0,0 +1,161 @@
+				>
+					<Field
+						type="password"
+						id="register-password-input"
+						name="password"
+						label="Password"
+						isRequired={true}
+						placeholder="Create a password"
+						autoComplete="current-password"
+						value={form.password}
+						onChange={handleChange}
+						error={errors.password && errors.password[0]}
```

Biome

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)